libethdrivers/src/plat/zynqmp/io.h

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
This seem a generic util lib that is not ZynqMP specific. Maybe move it up a bit, so it can also be used by other code?

Reply:
This was copied directly from the zynq7000 libethdrivers.

A future PR could reformat to a `mach` directory for the common code between the `zynqmp` and `zynq7000` ethdrivers. But I think a more incremental approach would be better here.